tiatoolbox/cli/overlay_from_json.py

```diff
@@ -0,0 +1,169 @@
+"""Command line interface for overlay_from_json."""
+from pathlib import Path
+import click
+
+from tiatoolbox.cli.common import (
+    cli_img_input,
+    cli_output_path,
+    prepare_file_dir_cli,
+    tiatoolbox_cli,
+)
+
+
+def numpy2vips(a):
+    import pyvips
+
+    dtype_to_format = {
+        "uint8": "uchar",
+        "int8": "char",
+        "uint16": "ushort",
+        "int16": "short",
+        "uint32": "uint",
+        "int32": "int",
+        "float32": "float",
+        "float64": "double",
+        "complex64": "complex",
+        "complex128": "dpcomplex",
+    }
+    height, width, bands = a.shape
+    linear = a.reshape(width * height * bands)
+    vi = pyvips.Image.new_from_memory(
```

Comment:
Can we not avoid pyvips dependancy? 
Adding it would mean further restrictions on dependency versions.

Reply:
Indeed tifffile / zarr cna be used instead here.

Reply:
I think this is resolved now.